boards/stk3200/include/board.h

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2015-2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_stk3200
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the STK3200 starter kit
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Bas Stottelaar <basstottelaar@gmail.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+#include "periph/gpio.h"
+#include "periph/spi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Xtimer configuration
+ *
+ * The timer runs at 250 KHz to increase accuracy.
+ * @{
+ */
+#define XTIMER_HZ           (250000UL)
+#define XTIMER_WIDTH        (16)
+/** @} */
+
+/**
+ * @name    Board controller configuration
+ *
+ * Define the GPIO pin to enable the BC, to allow serial communication
+ * via the USB port.
+ * @{
+ */
+#define BC_PIN              GPIO_PIN(PA, 9)
```

Comment:
```suggestion
#define BC_PIN              GPIO_PIN(PD, 7)
```

Reply:
That does make sense, according to the [manual](https://www.silabs.com/documents/public/user-guides/efm32zg-stk3200-ug.pdf). But it's strange that it does work for me. I'll try as well.

Reply:
It seems that this breaks RX for me.

Reply:
The schema is [contradicting](https://www.silabs.com/documents/public/schematic-files/EFM32ZG_BRD2010A_schematic.pdf)